_raml/apps/products/products.raml

```diff
@@ -74,6 +86,7 @@ get:
                 - /priceInfo/ecoParticipationPrice/amount<br>
                 - /priceInfo/depositPrice/amount<br>
                 - /productImage¹<br><br>
+                - /stocklevel<br><br>
```

Comment:
add footnote to stocklevel describing that its value will not be replaced but changed:
If the stocklevel is 3 and you "add" 1 you will  get 4. If you  add -5, you will get -2